packages/@coorpacademy-components/src/molecule/quick-access-card/index.js

```diff
@@ -77,17 +77,17 @@ const getBackgroudRadialColors = feature => {
 };
 
 const QuickAccessCard = (props, context) => {
-  const {title, description, feature, onClick, _blank} = props;
+  const {title, description, feature, href, _blank} = props;
   const {iconColor, degrees} = getBackgroudRadialColors(feature);
   const Icon = getIcon(feature);
-  const handleClick = useMemo(() => e => onClick(e), [onClick]);
   const newTabOnClick = _blank ? <NewTabIcon className={style.newTabIcon} /> : null;
 
   return (
-    <div data-name="card" data-type={feature} className={style.quickAccess} onClick={handleClick}>
-      <div
-        style={{
-          background: `radial-gradient(62.12% 56.45% at 0% 77.29%, 
+    <div data-name="card" data-type={feature} className={style.quickAccess}>
```

Comment:
```suggestion
    <div data-name="quick-access-card" data-type={feature} className={style.quickAccess}>
```
pour les tests d’intégration....